org.eclipse.xtext.common.types.tests/pom.xml

```diff
@@ -43,6 +43,10 @@
 			<groupId>org.eclipse.emf</groupId>
 			<artifactId>org.eclipse.emf.mwe2.language</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.eclipse.emf</groupId>
+			<artifactId>org.eclipse.emf.mwe2.launch</artifactId>
```

Comment:
Should this be in scope compile?

Reply:
as it is a test project i am not sure (is there a compileonly?)
this is why i asked @LorenzoBettini for a hint/pattern

Reply:
TIL there is no compileonly in mvn scopes.

Reply:
I think that's because there's no good reason to have a dependency only available at compile time ;) at least that's always been my understanding